.github/workflows/OCV-PR-5.x-ARM64.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         shell: bash
     container:
       image: quay.io/opencv-ci/opencv-ubuntu-20.04-arm64:20230413
+      options: --gpus all -m 8G
```

Comment:
Looks like --gpu option is not supported on ARM here and above:
```
Error response from daemon: could not select device driver "" with capabilities: [[gpu]]
```

Reply:
Please ignore audio test failure. It's know sporadic issue related to sound.

Reply:
I just checked available `display` or `VGA` devices on the ARM host and it turns out that the chip is not integrated with a GPU that supports OpenCL. We can set `-DWITH_OPENCL=OFF` for the ARM host.